lib/web_ui/lib/platform_dispatcher.dart

```diff
@@ -215,7 +223,7 @@ class ViewConfiguration {
     List<DisplayFeature>? displayFeatures,
   }) {
     return ViewConfiguration(
-      window: window ?? this.window,
+      view: view ?? this.view,
```

Comment:
assert that only one is provided and add

```dart
window: window ?? this.window,
```

Reply:
Resolved. See https://github.com/flutter/engine/pull/38453#discussion_r1061041530.